File: CascadingComboboBox/BranchCB.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CascadingComboboBox
{
    internal class BranchCB
    {
        public int BranchID { get; set; }
        public string BranchName { get; set; }

        public static BranchCB BlankBranch
        {
            get
            {
                return new BranchCB { BranchID = 0, BranchName = "" };
            }
        }

        public override string ToString()
        {
            return "BranchID: " + BranchID + " Name: " + BranchName;
        }
    }
}

File: CascadingComboboBox/BranchCB.vb

Namespace CascadingComboboBox
    Friend Class BranchCB
        Public Property BranchID As Integer
        Public Property BranchName As String

        Public Shared ReadOnly Property BlankBranch As BranchCB
            Get
                Return New BranchCB With {
                    .BranchID = 0,
                    .BranchName = ""
                }
            End Get
        End Property

        Public Overrides Function ToString() As String
            Return "BranchID: " & BranchID.ToString() & " Name: " & BranchName
        End Function
    End Class
End Namespace
